027-network-policy-generation-feature-gate.md

```diff
@@ -0,0 +1,63 @@
+# Network Policy Generation Feature Gate
+
+The feature would consist of a new Feature Gate which can be used to disable the generation of Network Policies by Strimzi, where the user wishes to set up a more restrictive Network Policy set.
```

Comment:
As mentioned below, I would prefer for this to talk about custom network policies rather than more strict policies. Based on previous issues, there will be users who will use this to disable network policies what so ever for example.

Reply:
As above, now using "custom".